release-notes/release-notes56.md

```diff
@@ -44,6 +44,11 @@ A new optional `name` field that sets a specific name to the format when it's be
 
 For information on the `name` field, see: [User interface - `style_formats`]({{ site.baseurl }}/configure/editor-appearance/#style_formats).
 
+### New `TableModified` event
+
+The new `TableModified` event is fired whenever style or structural changes are made to a table.
+
+For more information on the `TableModified` event, see: [Table - `events`]({{ site.baseurl }}/plugins/table/#events).
```

Comment:
Minor thing and doesn't affect the output but I think there should be an extra line separating the text from the next heading like the others.